flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStOptions.java

```diff
@@ -58,6 +58,22 @@ public class ForStOptions {
                                     "The remote directory where ForSt puts its SST files, fallback to %s if not configured.",
                                     LOCAL_DIRECTORIES.key()));
 
+    public static final ConfigOption<String> CACHE_DIRECTORY =
+            ConfigOptions.key("state.backend.forst.cache-dir")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            String.format(
+                                    "The directory where ForSt caches its SST files, fallback to %s/cache if not configured.",
+                                    LOCAL_DIRECTORIES.key()));
+
+    public static final ConfigOption<Long> CACHE_CAPACITY =
+            ConfigOptions.key("state.backend.forst.cache-capacity")
+                    .longType()
+                    .defaultValue(-1L)
+                    .withDescription(
+                            "The capacity capacity of cache, a negative value means no cache will be used. When this value is greater than the actual available space, the actual available space will be used as the upper limit.");
+
```

Comment:
How about 2 options:
1. state.backend.forst.cache.size-based-limit
2. state.backend.forst.cache.disk-remaining-space-based-limit

Reply:
`state.backend.forst.cache-capacity` is changed to `state.backend.forst.cache.size-based-limit`.
This option is for the `SystemSpaceChecker`, `SystemSpaceChecker` would pick the maximum of `size-based-limit` and disk space as the upper bound, I think one option is enough.

Reply:
@fredia What if there's more than one subtask per Taskmanager?